scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -105,7 +105,7 @@ def compute_config(args, cpu_flags=cpuflags(), machine=None):
         targetdevprefix = "sd"
         targetbus = "ide"
     controller_raid_volumes = args.controller_raid_volumes
-    if args.nodecounter != "1":
+    if args.nodecounter > args.numcontrollers:
```

Comment:
Once one parameter is converted as `int`, the other one needs to be converted too. For now I suggest to use `if int(args.nodecounter) > args.numcontrollers:`.

I am fixing the tests for we can refactor this.